package.json

```diff
@@ -83,7 +84,7 @@
     "mkdirp": "0.5.1",
     "mocha": "2.5.3",
     "phantomjs-polyfill-find": "ptim/phantomjs-polyfill-find",
-    "phantomjs-prebuilt": "2.1.7",
+    "phantomjs-prebuilt": "^2.1.14",
```

Comment:
Yeah, yarn actually inserted this caret, not I. So, I imagine we can phase them out as we upgrade packages in the release cycle upgradathon!